be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -361,7 +362,7 @@ class PipelineDriver {
     static constexpr int64_t YIELD_MAX_TIME_SPENT = 100'000'000L;
     // Yield PipelineDriver when maximum time in nano-seconds has spent in current execution round,
     // if it runs in the worker thread owned by other workgroup, which has running drivers.
-    static constexpr int64_t YIELD_PREEMPT_MAX_TIME_SPENT = 20'000'000L;
+    static constexpr int64_t YIELD_PREEMPT_MAX_TIME_SPENT = 5'000'000L;
```

Comment:
do 5ms make a difference more remarkable than 20ms?

Reply:
Yes, this could reduce the schedule latency for higher-priority workgroup preempting lower-priority workgroup.